dev/scripts/dbschema-generate.sh

```diff
@@ -22,7 +22,7 @@ set -euo pipefail
 SCRIPT_DIR="$(cd -P -- "$(dirname "$0")" && pwd -P)"
 ROOT_DIR="$(cd -P -- "${SCRIPT_DIR}/../../" && pwd -P)"
 MIGRATION_DIR="$(cd -P -- "${ROOT_DIR}/persistence-migration" && pwd -P)"
-CONTAINER_ID="$(docker run -d --rm -e 'POSTGRES_DB=dtrack' -e 'POSTGRES_USER=dtrack' -e 'POSTGRES_PASSWORD=dtrack' -p '5432' postgres:13-alpine)"
+CONTAINER_ID="$(docker run -d --rm -e 'POSTGRES_DB=dtrack' -e 'POSTGRES_USER=dtrack' -e 'POSTGRES_PASSWORD=dtrack' -p '5432' postgres:17-alpine)"
```

Comment:
13 is our current lowest supported version. All tests must work with it. Do you need a PG feature that is only available in newer versions?